go-controller/pkg/ovn/master.go

```diff
@@ -1067,14 +934,7 @@ func (oc *Controller) deleteNodeLogicalNetwork(nodeName string) error {
 }
 
 func (oc *Controller) deleteNode(nodeName string, hostSubnets []*net.IPNet) error {
```

Comment:
We could remove hostSubnets as an argument of the function

Reply:
Good catch. Makes the code simpler and less error prone too, like now we can clean up nodes that might have had their subnets annotation manually removed.